api/FileSystemHandle.json

```diff
@@ -109,7 +109,7 @@
       },
       "move": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FileSystemHandle/move",
+          "description": "<code>move</code>",
```

Comment:
We can actually just remove the description, since Yari auto-resolves to this description anyways:

```suggestion
```